src/scenes/home/informationForm/informationForm.js

```diff
@@ -16,6 +16,7 @@ class SignupInformation extends Component {
     super(props);
     this.onIdentifierStatusChange = this.onIdentifierStatusChange.bind(this);
     this.onCheckBoxChange = this.onCheckBoxChange.bind(this);
+    this.previousPage = this.previousPage.bind(this);
```

Comment:
It seems like we have to bind it if we want to know what is the current page in order to not going back if it's the first page